Calculator.Tests/Calculator.Tests/CalculatorTests.cs

```diff
@@ -0,0 +1,248 @@
+            Assert.AreEqual("0.00", actualInterest);
+
+        }
+
+        [Test]
+        public void PositiveTestInvestmentTermIsMandatoryField()
+        {
+            browser.FindElement(By.Id("amount")).SendKeys("100");
+            browser.FindElement(By.Id("percent")).SendKeys("10");
+            browser.FindElement(By.Id("term")).SendKeys(" ");
+            browser.FindElement(By.Id("d365")).Click();
+            string actualIncome = browser.FindElement(By.Id("income")).GetAttribute("value");
+            Assert.AreEqual("100.00", actualIncome);
+            string actualInterest = browser.FindElement(By.Id("interest")).GetAttribute("value");
+            Assert.AreEqual("0.00", actualInterest);
+
+        }
+
+        [Test]
+        public void PositiveTestStartDateIsMandatoryField()
+        {
+            browser.FindElement(By.Id("amount")).SendKeys("100");
+            browser.FindElement(By.Id("percent")).SendKeys("10");
+            browser.FindElement(By.Id("term")).SendKeys("365");
+            // NEED HELP couldn't manage this Test, unfortunately
+
+            SelectElement daySelect = new SelectElement(element:browser.FindElement(By.Id("day")));
+            string day = daySelect.SelectedOption.Text;
+            SelectElement monthSelect = new SelectElement(element: browser.FindElement(By.Id("month")));
+            string month = monthSelect.SelectedOption.Text;
+            SelectElement yearSelect = new SelectElement(element: browser.FindElement(By.Id("year")));
+            string year = yearSelect.SelectedOption.Text;
+            string actualDate = day + "/" + month + "/" + year;
+            string expectedDate = DateTime.Today.ToString("d/MMMM/yyyy");
+            Assert.AreEqual(expectedDate, actualDate);
+
+        }
+
+        [Test]
+        public void TestSelectAnyStart()
+        { 
+            SelectElement daySelect = new SelectElement(element: browser.FindElement(By.Id("day")));
+            
+            SelectElement monthSelect = new SelectElement(element: browser.FindElement(By.Id("month")));
+            
+            SelectElement yearSelect = new SelectElement(element: browser.FindElement(By.Id("year")));
+            // 2 April 2022
+
+            daySelect.SelectByText("2");
+            monthSelect.SelectByText("April");
+            yearSelect.SelectByText("2022");
+
+            string actualDate = daySelect.SelectedOption.Text + " " + monthSelect.SelectedOption.Text + " " + yearSelect.SelectedOption.Text;
+            Assert.AreEqual("2 April 2022", actualDate);
+
+        }
+        [Test]
+        public void TestFinancialYearIsMandatoryField()
+        {
+            
+            // NEED HELP how to skip required field properly? 
+            bool d365 = browser.FindElement(By.Id("d365")).Selected;
+            bool d360 = browser.FindElement(By.Id("d360")).Selected;
+
+            Assert.IsTrue(d365 || d360);
+            Assert.IsFalse(d365 && d360);
```

Comment:
You may add description to asserts to make failed tests easy to understand. These messages will be printed in test failed error.

```suggestion
            Assert.IsTrue(d365 || d360, "At least one option should be selected.");
            Assert.IsFalse(d365 && d360, "Only one option should be selected.");
```

Reply:
I have added those messages as //comments temporarily. The app underlines it. NEED HELP how to insert them as messages correctly. 

            Assert.IsTrue(d365 || d360); //"At least one option should be selected."
            Assert.IsFalse(d365 && d360); //"Only one option should be selected."